slash/exception_handling.py

```diff
@@ -83,7 +83,11 @@ def handling_exceptions(fake_traceback=True, **kwargs):
 
     if not PYPY and fake_traceback:
         # Only in CPython we're able to fake the original, full traceback
-        fake_tbs = create_traceback_proxy(frame_correction=2)
+        try:
+            fake_tbs = create_traceback_proxy(frame_correction=2)
+        except:
```

Comment:
Obviously, `except:` is far from being a good practice, but there's too many cases to handle when the underlying file had been changed (`IndexError` being not the only one..)